common/traits/000_wc_leadership_traits.txt

```diff
@@ -0,0 +1,195 @@
+# Written by: DukeJeans
+# Created: 1/22/2019
+# Modified: 3/28/2019
+#Leadership traits
+
+#Warcraft
+archmagus_leader = {
+	leader = yes
+	random = no
+	customizer = no
+
+	command_modifier = {
+		light_infantry = 0.2
```

Comment:
Use actual_light_infantry, light_infantry gives bonus to archers and light infanty

Reply:
>+20% to mages

Will someone take archmagus_leader instead of offensive_caster_leader or defensive_caster_leader?
I think not.

Reply:
I don't believe there's such an event that gives them a choice like that

Reply:
This is a problem of balance, not choice.
Archmage gives + 20% to mages
The attacking / defensive caster also gives +20% to mages AND other modifiers.
In 100 cases out of 100, the Attacking / Defensive caster will be better than the Archmage.

Reply:
Imo, attacking / defensive caster should give less bonus to mages than the archmage.
For example, 5% instead of 20%?

Reply:
Already way ahead of you :)